rudder-core/src/main/scala/com/normation/rudder/services/nodes/NodeInfoService.scala

```diff
@@ -83,7 +116,7 @@ trait NodeInfoService {
    * @param nodeId
    * @return
    */
-  def getNodeInfo(nodeId: NodeId) : Box[NodeInfo]
+  def getNodeInfo(nodeId: NodeId) : Box[Option[NodeInfo]]
```

Comment:
Why changing to box Option and not keeping the failure 'the Node {id} was not found' ?